### PR TITLE
feat: added breadcrumbs to pages

### PR DESCRIPTION
**Pull request by gilesdrinkall**
# Description

-   To add breadcrumbs to pages to help the user navigate around the site

# Testing instructions

-   The functionality can be tested locally via the fdbt-dev repo.

# Type of change

-   [X] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

# Checklist:

-   [X] Able to run pr locally
-   [X] Lighthouse performed (Accessibility 90+ minimum)
-   [X] Followed acceptance criteria
-   [X] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my own code
-   [ ] I have made corresponding changes to the documentation if applicable
-   [
Screenshot 2020-06-25 at 08 57 20
X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tests/utils/breadcrumbs.test.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    mockPeriodGeoZoneSeniorFromCsvZoneUploadBreadcrumbs,
+    mockFlatFareStudentFromDefinePassengerTypeBreadcrumbs,
+    mockMultiServicesAnyoneFromMultipleProductValidityPageBreadcrumbs,
+    mockMultiServicesAnyoneFromPeriodValidityPageBreadcrumbs,
+} from '../testData/mockData';
+
+describe('breadcrumbs', () => {
+    let ctx: NextPageContext;
+
+    it('creates the correct array of Breadcrumbs if user is on matching page having selected single, adult and csv upload', () => {
```
Comment (tejpowar) [inline tests/utils/breadcrumbs.test.ts:16]: Do we need to add a test if on home page?

File: src/utils/breadcrumbs.ts [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+    if (!url || url === '/') {
+        return {
+            generate: (): Breadcrumb[] => [],
+        };
+    }
+
+    const fareType = getCookieValue(ctx, FARE_TYPE_COOKIE, 'fareType');
+    const outboundJourney = getCookieValue(ctx, JOURNEY_COOKIE, 'outboundJourney');
+    const inputMethod = getCookieValue(ctx, INPUT_METHOD_COOKIE, 'inputMethod');
```
Comment (tejpowar) [inline src/utils/breadcrumbs.ts:23]: wanted to ask what the purpose of this cookie is?

File: tests/utils/breadcrumbs.test.ts (hunk 1 above)
Comment (gilesdrinkall) [reply tests/utils/breadcrumbs.test.ts:16]: Now added.

File: src/utils/breadcrumbs.ts (hunk 2 above)
Comment (gilesdrinkall) [reply src/utils/breadcrumbs.ts:23]: It's there so we know which input method the user selected to in turn show the correct breadcrumbs. It's used on both ln. 45 & 46.

**Pull request merged**